### PR TITLE
Docker Desktop 4.14.1 release notes

### PR DESCRIPTION
**Pull request by ebriney**
Signed-off-by: Emmanuel Briney <emmanuel.briney@gmail.com>

### Proposed changes

Add Docker Desktop 4.14.1 release notes

### CODE REVIEW

File: desktop/release-notes.md [hunk 1]
```diff
@@ -24,6 +24,23 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.14.1
+
+2022-11-16
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
```
Comment (mat007) [inline desktop/release-notes.md:33]: Should we not remove this from 4.14.0?

Comment (StefanScherer) [reply desktop/release-notes.md:33]: Also the `_includes/desktop-install.html` needs the new checksums in it.

Comment (ebriney) [reply desktop/release-notes.md]: just repushed

File: desktop/release-notes.md [hunk 2]
```diff
@@ -24,14 +24,61 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
 
 > Download Docker Desktop
 >
 > {% include desktop-install.html %}
 
+### Bug fixes and minor changes
+
+#### For all platforms
+
+- Fix container DNS lookups when using Registry Access Management.
```
Comment (aevesdocker) [inline desktop/release-notes.md:39]:
```suggestion
- Fixed container DNS lookups when using Registry Access Management.
```

File: desktop/release-notes.md [hunk 3]
```diff
@@ -24,14 +24,61 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+
+#### For Mac
+
+- Fixes an issue preventing "Analyze Image" button from Images screen to be working.
```
Comment (aevesdocker) [inline desktop/release-notes.md:43]:
```suggestion
- Fixed an issue preventing the **Analyze Image** button on the **Images** tab from working.
```

File: desktop/release-notes.md [hunk 4]
```diff
@@ -24,14 +24,61 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
...
+- Fixes a bug causing symlinks to not be created for the user if `/usr/local/lib` doesn't already exist. Fixes [docker/for-mac#6569](https://github.com/docker/for-mac/issues/6569)
```
Comment (aevesdocker) [inline desktop/release-notes.md:44]:
```suggestion
- Fixed a bug causing symlinks to not be created for the user if `/usr/local/lib` doesn't already exist. Fixes [docker/for-mac#6569](https://github.com/docker/for-mac/issues/6569)
```

**Review by aevesdocker — APPROVED** [review]
Thank you @ebriney

**Pull request merged**